### PR TITLE
 #168131404 Integrate TravisCI with readme badge

### PR DESCRIPTION
**Pull request by Bahatiroben**
#### What does this PR do?
- it sets up travis for continuous integration
#### Description of Task to be completed?
- the .travis.yml file includes necessary installs for the build to take place
- the test script is run and can either pass or fail
- upon pass, the merge is possible, otherwise not
#### How should this be manually tested?
- clone the repo
- branch off develop and make some changes
- push the changes
- you should see the travis notifications that some builds are taking place
#### Any background context you want to provide?
```
- for now, the repo is not connected to travis due to admin issue.
 therefore, the builds are not triggered
- as soon as travis is integrated, builds start.
```
#### What are the relevant pivotal tracker stories?
[#168131404](https://www.pivotaltracker.com/story/show/168131404)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  email: false
+cache:
+  directories:
+    - node_modules
+install:
+  - npm install
+services:
+  - postgresql
+before_script:
+  - psql -c 'drop database if exists barefoot_test;' -U postgres
```
Comment (Dkabiswa) [inline .travis.yml]: I thought we have to create a database in the before script rather than dropping it

**Review by Dkabiswa — CHANGES_REQUESTED** [review]
look into the before script

File: .travis.yml (hunk 1 above)
Comment (Bahatiroben) [reply .travis.yml]: I missed a line for creating It. I am adding it.

**Comment by drayzii** [comment]
Shouldn't we have a Travis check in the PR

**Comment by Bahatiroben** [comment]
> Shouldn't we have a Travis check in the PR

Andela admin have not yet confirmed the request I sent which was supposed to connect github to travis. So as soon as the admin confirms, Travis will start showing on every PR

**Review by Dkabiswa — APPROVED** [review]

**Review by drayzii — APPROVED** [review]

**Review by Paulvitalis200 — APPROVED** [review]

**Pull request merged**